### PR TITLE
Changing delete and remove user icon

### PR DESCRIPTION
**Pull request by talwag**

### CODE REVIEW

File: packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+} from "@material-ui/core";
+import { OverridableComponent } from "@material-ui/core/OverridableComponent";
+import Tooltip from "@material-ui/core/Tooltip";
+
+export enum IconSize {
+  small = "small",
+  medium = "medium",
+}
+
+export enum Icon {
```
Comment (yaron1m) [inline packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx:15]: Is this used?

File: packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import {
+  IconButton as MaterialIcon,
+  PropTypes,
+  SvgIconTypeMap,
+} from "@material-ui/core";
+import { OverridableComponent } from "@material-ui/core/OverridableComponent";
+import Tooltip from "@material-ui/core/Tooltip";
+
+export enum IconSize {
```
Comment (yaron1m) [inline packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx:10]: Is this used?

File: packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  "aria-label"?: string;
+  tooltipText?: string;
+}
+
+function MuiIconButton({ icon, tooltipText, ...props }: MuiIconButtonProps) {
+  const [IconComponent, setIconComponent] =
+    useState<OverridableComponent<SvgIconTypeMap<{}, "svg">>>();
+
+  useEffect(() => {
+    import("@material-ui/icons").then((icons) => setIconComponent(icons[icon]));
```
Comment (yaron1m) [inline packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx:35]: Why do we need this import?

File: packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  }, [icon]);
+
+  if (!IconComponent) return <></>;
```
Comment (yaron1m) [inline packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx:38]: It is better to return null

File: packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx (hunk 1 above)
Comment (talwag) [reply packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx:15]: This is the old component and it did use it. as we decided, for now we will keep it.

File: packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx (hunk 2 above)
Comment (talwag) [reply packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx:10]: This is the old component and it did use it. as we decided, for now we will keep it
But i did remove it from the new component now.

File: packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx (hunk 3 above)
Comment (talwag) [reply packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx:35]: This is the old component and it did use it. as we decided, for now we will keep it

File: packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx (hunk 4 above)
Comment (talwag) [reply packages/coordinator/src/components/MuiIconButton/MuiIconButton.tsx:38]: This is the old component and it did use it. as we decided, for now we will keep it

**Review by yaron1m — APPROVED** [review]

**Pull request merged**